examples/shell/shell_common/include/Globals.h

```diff
@@ -18,27 +18,59 @@
 #pragma once
 
 #include <lib/core/CHIPCore.h>
-#include <messaging/ExchangeMgr.h>
-#include <protocols/secure_channel/MessageCounterManager.h>
-#include <transport/SecureSessionMgr.h>
+#include <stack/Stack.h>
 #include <transport/raw/TCP.h>
-#include <transport/raw/UDP.h>
+
+constexpr size_t kMaxPayloadSize            = 1280;
+constexpr size_t kResponseTimeOut           = 1000;
+constexpr chip::Transport::AdminId gAdminId = 0;
 
 #if INET_CONFIG_ENABLE_TCP_ENDPOINT
-constexpr size_t kMaxTcpActiveConnectionCount = 4;
-constexpr size_t kMaxTcpPendingPackets        = 4;
+
+class TransportConfigurationWithTcp : chip::TransportConfiguration
+{
+public:
+    static constexpr size_t kMaxTcpActiveConnectionCount = 4;
+    static constexpr size_t kMaxTcpPendingPackets        = 4;
+
+    using transport = chip::TransportMgr<
```

Comment:
nit: I think we generally start type names with a capital letter. Change to "Transport"?